modules/claims_api/app/swagger/claims_api/v1/form_0966_controller_swagger.rb

```diff
@@ -146,6 +146,7 @@ class Form0966ControllerSwagger
                     end
                     property :participant_claimant_id do
                       key :type, :integer
+                      key :required, :false
```

Comment:
should not be required if we are auto loading claimant ID from @current_user above.